lib/modules/manager/gradle-wrapper/utils.ts

```diff
@@ -61,6 +66,29 @@ export function getJavaConstraint(
   return '^11.0.0';
 }
 
+// https://docs.gradle.org/current/userguide/gradle_daemon.html#sec:daemon_jvm_criteria
+export async function getJvmConfiguration(
+  gradlewFile: string,
+): Promise<string | null> {
+  const daemonJvmFile = join(
+    dirname(gradlewFile),
+    'gradle/gradle-daemon-jvm.properties',
+  );
+  const daemonJvm = await readLocalFile(daemonJvmFile, 'utf8');
+  if (daemonJvm) {
+    const TOOLCHAIN_VERSION_REGEX = regEx(
+      '^(?:toolchainVersion=)(?<version>\\d+)$',
```

Comment:
I have no reason to believe this to be a major issue, but if the properties file ever looks like
```
toolchainVersion = 11
```
or any combination of whitespace around `'='`, this regex will fail to match it.

Reply:
Thanks for fixing it!